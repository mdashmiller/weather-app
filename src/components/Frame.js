import styled from "styled-components"

const Frame = styled.div`
	width: 300px;
	margin: 40px auto 30px;
	text-align: center;
	font-family: 'Varela Round', Arial, sans-serif;
`

export default Frame
