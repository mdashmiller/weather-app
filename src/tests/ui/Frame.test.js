import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Frame from '../../components/ui/Frame'

describe('<Frame> rendering', () => {

	it('renders without crashing', () => {
	  	const div = document.createElement('div')
	  	ReactDOM.render(<Frame>Test</Frame>, div)
	  	ReactDOM.unmountComponentAtNode(div)
	})

	test('has a valid snapshot', () => {
		const component = renderer.create(
			<Frame>Test</Frame>
		)
		let tree = component.toJSON()
		expect(tree).toMatchSnapshot()
	})

})
