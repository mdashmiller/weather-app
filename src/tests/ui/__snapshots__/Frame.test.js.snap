// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Frame> rendering has a valid snapshot 1`] = `
<div
  className="sc-bdVaJa cOdhsb"
>
  Test
</div>
`;
